<launch>
	<node
		pkg="grabby_mcarmface_ros"
		type="serialNode"
		name="serialNode"
		output="screen"
		respawn="false"
		>
	<!-- Parameter Set -->
	<rosparam file="$(find grabby_mcarmface_ros)/params/serialNode.yaml" />
	</node>
</launch>